extension.js

```diff
@@ -240,22 +348,22 @@ class ReferenceProvider {
                     let continueFrom;
 
                     let dotPosition = range._start._character >= 1 ?
-                                        document.offsetAt(new vscode.Position(range._start._line, range._start._character-1)) :
+                                        document.offsetAt(new Position(range._start._line, range._start._character-1)) :
                                         0;
 
                     //Do backwards search for a dot
-                    dotPosition = this.doBackwardsSearch(fullText, dotPosition, ".")
-                    const haveParent = dotPosition !== false;
+                    dotPosition = this.doBackwardsSearch(fullText, dotPosition, ".");
+                    const hasParent = dotPosition !== false;
 
                     let tmpModuleName;
-                    if(!haveParent) {
-                        tmpModuleName = this.extractString(document, range)
+                    if (!hasParent) {
+                        tmpModuleName = this.extractString(document, range);
                     }
 
-                    const constructors = this.findConstructor(document, textAtCaret, fullText);
+                    const constructors = this.findConstructor(document, textAtCaret, textToParse);
```

Comment:
Fixed the bug in stringIsPartOfDefineOrRequireStatement. For the error that shows up there is a separate issue: #22